resources/build/zip.inc.sh

```diff
@@ -115,3 +117,20 @@ function add_zip_files() {
     rm "${EXCLUDE_FILE}"
   fi
 }
+
+_verify_input() {
```

Comment:
This is a very ambiguously named function that could cause clashes in the future. Better would be `_zip_verify_input`.